spark/src/main/java/org/opensearch/sql/spark/asyncquery/model/AsyncQueryJobMetadata.java

```diff
@@ -59,6 +59,6 @@ public static AsyncQueryJobMetadata copy(
 
   @Override
   public String getId() {
-    return queryId.docId();
+    return "qid" + queryId;
```

Comment:
why add "qid"? is this for OpenSearch doc only?

Reply:
It is to keep the same logic.
It was originally implemented in AsyncQueryId class.